src/Docfx.App/PdfBuilder.cs

```diff
@@ -217,6 +217,13 @@ static string ExpandTemplate(string? pdfTemplate, int pageNumber, int totalPages
                     .Replace("<span class='totalPages'></span>", $"<span>{totalPages}</span>")
                     .Replace("<span class=\"totalPages\"></span>", $"<span>{totalPages}</span>");
             }
+
+            static string? GetHeaderFooter(string? template)
+            {
+                return File.Exists(template)
+                    ? File.ReadAllText(template)
```

Comment:
Where is the current directory if this is a relative path? Consider explicitly state the base directoy to relative to